app/World.php

```diff
@@ -947,6 +947,12 @@ public function setOverridePatch(array $patch): void
      */
     public function writeToRom(Rom $rom, bool $save = false): Rom
     {
+        if ($override_start_screen = $this->config('override_start_screen', false)) {
```

Comment:
I'm not 100% sure if this is the best place to be doing validation.  I'd be happy to hear a better idea.